release-notes/DxMs/cloudgateway_change_log.md

```diff
@@ -4,6 +4,30 @@ uid: cloudgateway_change_log
 
 # Cloud Gateway change log
 
+#### 29 January 2025 - Enhancement - CloudGateway 2.17.1 - Access tokens instantly refreshed when cloud session is manually renewed [ID 42081]
+
+From now on, when the [cloud session is renewed](xref:Cloud_Connection_Issues#check-the-cloud-session) manually, other access tokens, for example used by STaaS, will be refreshed instantly. Previously, this could take up to 30 minutes.
+
+#### 29 January 2025 - Enhancement - CloudGateway 2.17.1 - Offloading of local setting for Remote Access and Live Sharing [ID 42080]
+
+Cloud Gateway will now offload [the local setting for Remote Access and Live Sharing](xref:Disabling_Remote_Access_and_Live_Sharing).
```

Comment:
Could we also add the reason why this is done here? I think users will wonder about this.

Reply:
Same as below:
This allows for better technical support and for dataminer.services to validate if everything is correctly configured to use the DataMiner Cube desktop app remotely via dataminer.services.